src/base/system.h

```diff
@@ -997,20 +997,21 @@ void str_append(char *dst, const char *src, int dst_size);
 void str_copy(char *dst, const char *src, int dst_size);
 
 /*
-	Function: str_num_copy
-		Copies first num characters of a string to another.
+	Function: str_truncate
+		Truncates a string to a given length.
 
 	Parameters:
 		dst - Pointer to a buffer that shall receive the string.
-		src - String to be copied.
-		num - Number of characters to copy.
 		dst_size - Size of the buffer dst.
+		strc - String to be truncated.
```

Comment:
```suggestion
		src - String to be truncated.
```

Apparently forgot to press the comment button.